tests/unit/metadata/test_multi_table.py

```diff
@@ -435,3 +438,99 @@ def test_visualize_show_table_details_only(self, visualize_graph_mock):
             ('users', 'payments', '')
         ]
         visualize_graph_mock.assert_called_once_with(expected_nodes, expected_edges, 'output.jpg')
+
+    def test_add_column(self):
+        """Test the ``add_column`` method.
+
+        The method should get the appropriate table and call ``add_column`` on it.
+
+        Setup:
+            - Set the ``_tables`` attribute to have a mock for the table name.
+
+        Input:
+            - table_name that matches what is in ``_tables``.
+            - column_name.
+            - Some key word arguments.
+
+        Side effect:
+            - The mock should have the ``add_column`` method called with the right attributes.
+        """
+        # Setup
+        metadata = MultiTableMetadata()
+        table = Mock()
+        metadata._tables = {'table': table}
+
+        # Run
+        metadata.add_column('table', 'column', sdtype='numerical', pii=False)
+
+        # Assert
+        table.add_column.assert_called_once_with('column', sdtype='numerical', pii=False)
```

Comment:
If it's easy, I'd also check that metadata._tables was updated properly (since that's the actually only important part).

Reply:
Oh, you are patching the whole thing, I hadn't noticed. Then it's fine to leave as is (I just usually like to double check the values themselves just to be sure it actually worked, but only if it's easy.).

Reply:
yeah it wouldn't be doable here. The `MultiTableMetadata` just stores a dict containing `SingleTableMetadata` objects, so the column gets added at that level